SteamBot/Bot.cs

```diff
@@ -336,6 +364,24 @@ public void HandleBotCommand(string command)
             }
         }
 
+        public void HandleInput(string input)
+        {
+            consoleInput = input;
+        }
+
+        public string WaitForInput()
+        {
+            consoleInput = null;
+            while (true)
+            {
+                if (consoleInput != null)
+                {
+                    return consoleInput;
+                }
+                Thread.Sleep(5);
+            }
+        }
```

Comment:
Alright, we can review that later and possibly the other place it occurs in
the code base after, if its that  important.
On 2 Dec 2015 11:20 p.m., "Joel" <notifications@github.com> wrote:

> In SteamBot/Bot.cs
> <https://github.com/Jessecar96/SteamBot/pull/853#discussion_r46491992>:
>
> > +        {
> > +            consoleInput = input;
> > +        }
> > +
> > +        public string WaitForInput()
> > +        {
> > +            consoleInput = null;
> > +            while (true)
> > +            {
> > +                if (consoleInput != null)
> > +                {
> > +                    return consoleInput;
> > +                }
> > +                Thread.Sleep(5);
> > +            }
> > +        }
>
> Feel free to make a pull request later if you feel it's necessary, but I
> think it's important that we get mobile auth working with SteamBot ASAP
> first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Jessecar96/SteamBot/pull/853/files#r46491992>.
>